src/assets/data/markdown/ionic-weather-step02.md

```diff
@@ -56,12 +56,14 @@ Add at least the platform matching the device you will use for this course. If y
 1. run at least one of the following
     1. `ionic cordova platform add ios`
     1. `ionic cordova platform add android`
-1. `git add resources/*`
```

Comment:
You might think so, but I this is needed.

After `ionic start`:

```
~/Projects/Training/test (master): git sta
## master
```

After installing the platforms:

```
~/Projects/Training/test (master *): git sta
## master
 M config.xml
 M package-lock.json
 M package.json
 M resources/ios/icon/icon-1024.png
 M resources/ios/icon/icon-40.png
 M resources/ios/icon/icon-40@2x.png
 M resources/ios/icon/icon-40@3x.png
 M resources/ios/icon/icon-50.png
 M resources/ios/icon/icon-50@2x.png
 M resources/ios/icon/icon-60.png
 M resources/ios/icon/icon-60@2x.png
 M resources/ios/icon/icon-60@3x.png
 M resources/ios/icon/icon-72.png
 M resources/ios/icon/icon-72@2x.png
 M resources/ios/icon/icon-76.png
 M resources/ios/icon/icon-76@2x.png
 M resources/ios/icon/icon-83.5@2x.png
 M resources/ios/icon/icon-small.png
 M resources/ios/icon/icon-small@2x.png
 M resources/ios/icon/icon-small@3x.png
 M resources/ios/icon/icon.png
 M resources/ios/icon/icon@2x.png
 M resources/ios/splash/Default-568h@2x~iphone.png
 M resources/ios/splash/Default-667h.png
 M resources/ios/splash/Default-736h.png
 M resources/ios/splash/Default-Landscape-736h.png
 M resources/ios/splash/Default-Landscape@2x~ipad.png
 M resources/ios/splash/Default-Landscape@~ipadpro.png
 M resources/ios/splash/Default-Landscape~ipad.png
 M resources/ios/splash/Default-Portrait@2x~ipad.png
 M resources/ios/splash/Default-Portrait@~ipadpro.png
 M resources/ios/splash/Default-Portrait~ipad.png
 M resources/ios/splash/Default@2x~iphone.png
 M resources/ios/splash/Default@2x~universal~anyany.png
 M resources/ios/splash/Default~iphone.png
?? resources/icon.png.md5
?? resources/splash.png.md5
```

So you either need what I had above, or two commands to add each explicitly (which "best practices" wise is certainly better, but a little overkill in this case).

I suggest restoring this line.

Reply:
Happy to restore it - I just want to make sure I haven't been wrong this whole time o_o

Reply:
From the `commit` manpage:

```
OPTIONS
       -a, --all
           Tell the command to automatically stage files that have been
           modified and deleted, but new files you have not told Git about are
           not affected.
```

Reply:
Actually, using `-a` is considererd an anti-pattern in the hard-core `git` community, but I use it when convenient... lol

Reply:
FWIW, the hard-core `git` commuity "proper" way is to use `git stage --patch` to review all of your individual mods before staging and either add or reject them, explicitely add individual files via `git add` but never using `*`, then use `git commit` to launch `vi` and enter a good subject with perhaps some supporting description.